openwisp_monitoring/db/backends/elasticsearch/tests.py

```diff
@@ -0,0 +1,18 @@
+# from datetime import date, timedelta
+
+# from django.conf import settings
+# from django.core.exceptions import ValidationError
+# from django.test import TestCase
+# from django.utils.timezone import now
+# from openwisp_monitoring.device.settings import SHORT_RETENTION_POLICY
+# from openwisp_monitoring.device.utils import SHORT_RP, manage_short_retention_policy
+# from openwisp_monitoring.monitoring.tests import TestMonitoringMixin
+# from swapper import load_model
+
+# from .. import timeseries_db
+
+# Chart = load_model('monitoring', 'Chart')
+
+
+# class TestDatabaseClient(TestMonitoringMixin, TestCase):
+#     pass
```

Comment:
:eyes:

Reply:
I was working on having a Test class which is easily switchable between `influxdb` and `elasticsearch`, didn't push the latest changes.